packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -87,11 +99,16 @@ export default createConnector({
       console.warn('react-instantsearch: `searchForFacetValues` has been renamed to' +
         '`withSearchBox`, this will break in the next major version.');
     }
+    if (props.withSearchBox && this.context.multiIndexContext) {
+      // eslint-disable-next-line no-console
+      console.error('react-instantsearch: searching in *List is not available when used inside a' +
```

Comment:
Is a console.error sufficient or should we throw an error since it's no gonna work in the end? Here the code will try to go further.